src/classes/base/search-results.ts

```diff
@@ -0,0 +1,35 @@
+import dlv from 'dlv';
+import { mapTagTypeToUrl } from '../../utils/map-tag-type-to-url';
+import { StrapiAuthor } from '../strapi-author';
+import { formatRelatedContent } from '../../utils/format-related-content'
+
+
+export class SearchResult {
+    description: string
+    mediaType: string;
+    publishDate: string;
+    slug: string;
+    title: string;
+    tags: object[];
+    products: object[];
+    languages: object[];
+    authors: object[];
+    related: object[];
```

Comment:
Related is only part of each content, for search result I don't give back related.